### PR TITLE
Numpy update

### PR DESCRIPTION
**Pull request by kamleshkc2002**
Updating the installation instructions for various scientific formulae.

Relevant issues: https://github.com/sb2nov/mac-setup/issues/277

### CODE REVIEW

File: Python/numpy.md [hunk 1]
```diff
@@ -1,22 +1,19 @@
 # Numpy, Scipy and Matplotlib
```
Comment (hugovk) [inline Python/numpy.md:1]:
```suggestion
# NumPy, SciPy and Matplotlib
```

File: Python/numpy.md [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
 
-The [Numpy](http://numpy.scipy.org/), [Scipy](https://www.scipy.org/scipylib/index.html) and [Matplotlib](http://matplotlib.org/) scientific libraries for Python are always a little tricky to install from source because they have all these dependencies they need to build correctly. Luckily for us, [Samuel John](http://www.samueljohn.de/) has put together some [Homebrew formulae](https://github.com/samueljohn/homebrew-python) to make it easier to install these Python libraries.
+The [Numpy](http://numpy.scipy.org/), [Scipy](https://www.scipy.org/scipylib/index.html) and [Matplotlib](http://matplotlib.org/) scientific libraries for Python are always a little tricky to install from source because they have all these dependencies they need to build correctly.
```
Comment (hugovk) [inline Python/numpy.md:3]: Also NumPy and SciPy here.

Let's update the URLs too:
https://numpy.org/
https://matplotlib.org/

File: Python/numpy.md [hunk 3]
```diff
@@ -1,22 +1,19 @@
...
 
-    brew tap samueljohn/python
-    brew tap brewsci/bio
+* Using `pip`
 
-Then, install the `gfortran` dependency which we will need to build the libraries along with gcc:
+Python provides an inbuilt package management system `pip` which can be used to install these libraries
 
-    brew install gcc
+```shell script
```
Comment (hugovk) [inline Python/numpy.md:11]: Use `bash` or `sh`, `shell script` isn't an [option](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting)
```bash
```shell script
```

File: Python/numpy.md [hunk 4]
```diff
@@ -1,22 +1,19 @@
...
+* Using [Macports](https://www.macports.org/) and Python 3.5
 
-    pip install nose pyparsing python-dateutil pep8
-
-Finally, you can install Numpy and Scipy with:
-
-    brew install numpy scipy matplotlib
-
-(It may take a few minutes to build)
+```shell script
```
Comment (hugovk) [inline Python/numpy.md:17]:
```bash
```shell script
```

File: Python/numpy.md [hunk 5]
```diff
@@ -1,22 +1,19 @@
...
-Then, install the `gfortran` dependency which we will need to build the libraries along with gcc:
+Python provides an inbuilt package management system `pip` which can be used to install these libraries
 
-    brew install gcc
+```shell script
+python -m pip install --user numpy scipy matplotlib ipython jupyter pandas sympy nose
+```
 
-Then, install `nose`, `pyparsing` and `python-dateutil` dependency which we will need to build the libraries:
+* Using [Macports](https://www.macports.org/) and Python 3.5
```
Comment (hugovk) [inline Python/numpy.md:15]:
```suggestion
* Using [MacPorts](https://www.macports.org/) and Python 3.5
```

---

Can only Python 3.5 be used with MacPorts?

File: Python/numpy.md (hunk 1 above)
Comment (kamleshkc2002) [reply Python/numpy.md:1]: great callout. Added.

File: Python/numpy.md (hunk 2 above)
Comment (kamleshkc2002) [reply Python/numpy.md:3]: updated.

File: Python/numpy.md (hunk 3 above)
Comment (kamleshkc2002) [reply Python/numpy.md:11]: updated. (I blame intellij :) )

File: Python/numpy.md (hunk 4 above)
Comment (kamleshkc2002) [reply Python/numpy.md:17]: updated

File: Python/numpy.md (hunk 5 above)
Comment (kamleshkc2002) [reply Python/numpy.md:15]: These instructions are directly from the sciPy website. (https://www.scipy.org/install.html). I personally have not tested with scientific packages with updated versions of Python. I followed these instructions though and could install these on my mac. Also updated the Typo.

**Comment by musicalshore** [comment]
Please unsubscribe me from this fucking list.

**Comment by hugovk** [comment]
> Please unsubscribe me from this fucking list.

@musicalshore Please unwatch the repo if you don't want to be notified:

image

https://github.com/sb2nov/mac-setup/watchers?page=5

https://help.github.com/en/github/managing-subscriptions-and-notifications-on-github/managing-your-subscriptions?algolia-query=unwatch#unwatch-a-repository

**Comment by kamleshkc2002** [comment]
gentle nudge @hugovk

**Pull request merged**